### DISCUSSION TITLE
Custom board support in the source

### DISCUSSION THREAD

**Discussion by PontusO** [gab-k/RP2040-Decoder#25] [General]
Hey there,
Wev've made a derivative of your board (https://ilabs.se/product/opendec02) with a few modifications to the hw. With this change we have added a board configuration macro needed to handle the different GPIO pins that we have assigned.
Would you be interested in a PR merging these changes into you repo or would you rather we have this on a separate fork?  Also how can we support your efforts best?

**Comment by gab-k** [comment]
Hey, sorry for the late answer, as for the changes I would prefer to not merge those in.

**Comment by longissor** [comment]
Hi,

Did you keep the same regulator or did you change it? 
Do you have issues with heat from the regulator with DCC station using 16-18V DCC ?

Thank for your feedback.

**Reply by PontusO** [reply]
Not sure if this was a question for me but yes, the new version of our boards uses a switch mode supply which helps the boards to run much cooler. @gab-k  Hit me up if you want me to send over a couple of units to you.

**Reply by gab-k** [reply]
Hi @PontusO,
is there some way i can contact you (by email for example)?
Thank you
Gabriel

**Reply by PontusO** [reply]
Hi Gabriel,
I don't want to publish it here but if you head over to [ilabs.se](https://ilabs.se/contact-form/) and send me a note with your email I will reach out to you.
/Pontus

**Reply by longissor** [reply]
Thank you @PontusO 
switch mode is the best solution. Bit more complex. I am waiting to receive bigger linear regulators. This will help me progress for now
